core/config.go

```diff
@@ -21,6 +21,7 @@ const (
 	MajorLevel = "major"
 	MinorLevel = "minor"
 	PatchLevel = "patch"
+	NoneLevel  = "none"
```

Comment:
I think it will help us with readability to create an "EmptyLevel" as just an empty string. 
```suggestion
	NoneLevel  = "none"
	EmptyLevel = ""
```